input_writer.go

```diff
@@ -0,0 +1,30 @@
+package logrus
+
+import (
+	"bufio"
+	"io"
+	"runtime"
+)
+
+func (logger *Logger) InputWriter() (*io.PipeWriter) {
```

Comment:
Hm, I'm not sure I understand why `InputWriter`? Why `Input`? How do you feel it was unclear if it was just called `Writer`?

```go
log = logrus.New()
log.Writer()
# or
logrus.Writer() # if we expose at package level, which makes sense
```